App/public/css/fontcolor.css

```
@@ -0,0 +1,8 @@
+:root{
+    /* Colors */
+    --purple: #38256C;
+    --white-purple: #D6CEEC;
+
+    /* Font colors */
+    --slate-text-color: #606060;
```

Dans ce fichier essaie de ne pas associer une couleur à un usage 👍 
```suggestion
    --slate: #606060;
```

---

N.b: si dans le futur tu as besoin de variante, tu pourras utliser la méthode `color-{saturation}` avec satuation allant de 100 à 900 👍